### PR TITLE
MOSIP-31558, MOSIP-31523 :Added testcase for reset password sign up s…

### PR DESCRIPTION
**Pull request by nihallakshmankonchady**
…ervices. Get csrf testcase

### CODE REVIEW

File: Esignet/scripts/Esignet_Test_Script.jmx
Comment (Sanjayss04) [inline Esignet/scripts/Esignet_Test_Script.jmx:1]: Under thread group Sign Up Service - Reset Password (Preparation) - For all the http request the server IP variable name is used as "serverIPCamdgcPerf" which is not defined in the user defined variable. 
Replace it with "serverIP".

Comment (Sanjayss04) [inline Esignet/scripts/Esignet_Test_Script.jmx:1]: For storing the test data to csv for reset password, JSR223 post processor is been used with few lines of code whereas in other preparation thread groups for storing the test data we are using a bean shell post processor with StringToFile function. 
Can't we use that here ?

Comment (Sanjayss04) [inline Esignet/scripts/Esignet_Test_Script.jmx:1]: For storing the test data to csv for reset password in JSR223 post processor don't use such hard coded path "D:/apache-jmeter-5.3/bin/Testdataforresetpassword.csv".

You can either give "/apache-jmeter-5.3/bin/Testdataforresetpassword.csv". So it is understandable that we have to store the file in jmeter bin folder.

Comment (Sanjayss04) [inline Esignet/scripts/Esignet_Test_Script.jmx:1]: In "Sign Up Service - Reset Password (Execution)" thread group under Load identifier From File remove the path from csv data set config and don't browse it and keep it as hard coded. 
Keep it like this "./Testdataforresetpassword.csv" similar to all other places.

Comment (Sanjayss04) [inline Esignet/scripts/Esignet_Test_Script.jmx:1]: In "Sign Up Service - Reset Password (Execution)" thread group the 2 variables are not in proper naming convention "p_identifier" and "p_encodeddata".

Keep it something like "identifierPwd" or "identifierPassword". Please change for both the variables.

Comment (Sanjayss04) [inline Esignet/scripts/Esignet_Test_Script.jmx:1]: In "Sign Up Service - Reset Password (Execution)" thread group the sampler naming convention is not correct for Reset password - post request. Remove the post request which is added in the suffix. 
Replace it with "Reset Password Endpoint API".

Comment (Sanjayss04) [inline Esignet/scripts/Esignet_Test_Script.jmx:1]: In "Sign Up Service - Reset Password (Execution)" thread group under Reset password - post request sampler 2 pre processors are disabled "Generate Random Password" and "Generate UTC Time". If they are disabled and not in use please remove those.

Comment (nihallakshmankonchady) [reply Esignet/scripts/Esignet_Test_Script.jmx:1]: Here we are creating encoded value of JSON value of the Khmer names. So we could not call the StringToFile function.

Comment (nihallakshmankonchady) [reply Esignet/scripts/Esignet_Test_Script.jmx:1]: updated

Comment (nihallakshmankonchady) [reply Esignet/scripts/Esignet_Test_Script.jmx:1]: Updated

Comment (nihallakshmankonchady) [reply Esignet/scripts/Esignet_Test_Script.jmx:1]: Updated

Comment (nihallakshmankonchady) [reply Esignet/scripts/Esignet_Test_Script.jmx:1]: Updated

Comment (nihallakshmankonchady) [reply Esignet/scripts/Esignet_Test_Script.jmx:1]: Updated

Comment (nihallakshmankonchady) [reply Esignet/scripts/Esignet_Test_Script.jmx:1]: Removed the disabled components

**Comment by Sanjayss04** [comment]
@nihallakshmankonchady 
The branch is showing some conflicts, please review and check once.

**Comment by Sanjayss04** [comment]
The jmx file is not opening and throwing error. Pls check.
image

**Comment by nihallakshmankonchady** [comment]
Closing the PR due to the multiple conflicts.

**Pull request closed without merging**